### PR TITLE
Added cvpia_access argument to add_access() function

### PR DESCRIPTION
**Pull request by liz-fw**
cvpia_access = TRUE (default) returns write access for cvpia. I added documentation to the function and ran it in an EML package where it produced no errors and what looked like a valid xml.

### CODE REVIEW

File: R/access-element.R [hunk 1]
```diff
@@ -25,15 +25,21 @@
...
+#'   
 #' @return Returns an list with all the information required for the access section 
 #' of an EML document.
 #' @examples
 #' add_access()
 #' add_access(allow_principal = "private",
-#'            allow_permission = "none")
+#'            allow_permission = "none"
+#'            cvpia_acceess = FALSE)
 #' @export
```
Comment (ErinCain) [inline R/access-element.R:43]: please update tests to pass with new expected add_access outputs()

File: R/access-element.R [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #' @title Add Access Element
```
Comment (ErinCain) [inline R/access-element.R:1]: Run devtools::document() to make sure that new man file is generated

File: R/access-element.R [hunk 3]
```diff
@@ -25,15 +25,21 @@
...
 #'   * 'all' - all of the above.  
 #'   
+#' @param cvpia_access The cvpia_access argument allows for more than one set of permissions
+#' and takes a logical TRUE or FALSE value. With no input the default is TRUE, giving
+#' public "read" access and a CVPIA user group "write" access. If FALSE, the inputs
+#' for allow_permission and allow_principal are used as the single set of permissions.
+#'   
 #' @return Returns an list with all the information required for the access section 
 #' of an EML document.
 #' @examples
```
Comment (ErinCain) [inline R/access-element.R:36]: look into why example is not showing up on pkgdown site.

File: R/access-element.R (hunk 1 above)
Comment (liz-fw) [reply R/access-element.R:43]: updated add_access test in test-eml-elements.R

**Comment by liz-fw** [comment]
@ErinCain I had to add some functions to the pkgdown.yml file in order to rebuild the site and check that the add_access element function worked (generate_report_df, word_example, create_other_entity, add_other_entity). Just flagging that this is in one of the commits here, can change if it's a problem!

**Review by ErinCain — COMMENTED** [review]
A couple of things to follow up on but overall looks good!

**Review by ErinCain — APPROVED** [review]

**Pull request merged**